examples/nips17_adversarial_competition/README.md

```diff
@@ -0,0 +1,230 @@
+
+# Development toolkit for participants of adversarial competition
+
+NOTE: Entire content of this directory
+[http://google3/learning/brain/research/red_team/adversarial_competition/dev_toolkit]
+is intended to be published online and to be run outside of Google3.
+Thus it does not contain build files or any google3 dependencies.
+Text of this NOTE to be removed upon publishing.
+
+[TOC]
+
+This is a development toolkit for
+[Adversarial Attacks and Defences competition](https://www.kaggle.com/nips-2017-adversarial-learning-competition)
```

Comment:
name should be "Competition on Adversarial Examples and Defenses" to match the website